### DISCUSSION TITLE
Feature: New easter egg name: Rambo

### DISCUSSION THREAD

**Discussion by deurklink** [OpenRCT2/OpenRCT2#15797] [Feature requests]
**OS:** Windows 10
**Version:** 0.2.1
**Commit/Build:** 5e39147

Since some of the easter egg names were disabled in OpenRCT2, how about we replace one of them with this?

https://www.youtube.com/watch?v=FsIuXb2DMO8

Skip to 2:07 for the result.
I am using an unused peep flag for it, and the code that had to be added is all outside other functions, so it should not make the code any messier. I think many people would enjoy this being added to the game, but I can understand if this should be implemented as a plug-in instead 😉

**Comment by AaronVanGeffen** [comment]
> but I can understand if this should be implemented as a plug-in instead 😉

Yeah, I find myself leaning that way, too. 😉

**Comment by janisozaur** [comment]
Duplicate of https://github.com/OpenRCT2/OpenRCT2/pull/6190

**Comment by janisozaur** [comment]
Uh, didn't mean to close this really, clicked the wrong button. Feel free to reopen if needed.

**Comment by deurklink** [comment]
I can't reopen this myself.. It is something I would really like to implement some time in the future
